commands.pm

```diff
@@ -206,6 +216,8 @@ sub run_daemon {
     my $secret = $bmwqemu::vars{'CONNECT_PASSWORD'} || 'notsosecret';
     app->secrets([$secret]);
 
+    push @{app->static->paths}, '/';    # we check the path in each method
```

Comment:
Setting 2 paths (data, assets) could cause conflicts for files with the same name because the files are always searched in all paths.
So IHMO it is better to set common parent directory and because the paths can be arbitrary (they are set by worker), the only known common parent directory is /

It is as secure as before when the files were just read and sent.